cedar-policy-generators/src/settings.rs

```diff
@@ -55,6 +55,9 @@ pub struct ABACSettings {
     /// Flag to globally enable/disable generation of expressions containing the
     /// `like` operator.
     pub enable_like: bool,
+    /// Flag to globally enable/disable generation of expressions containing the
+    /// `is` operator.
+    pub enable_is: bool,
```

Comment:
Do we need this flag? `is` should be enabled for all targets?

Reply:
for the moment, yes. We're not modeling it in dafny, so I need to turn it off for difftesting

Reply:
Then I propose we update and merge it once the definitional model is done otherwise this PR essentially does nothing.

Reply:
I need to turn it off for difftesting, but not property testing on the Rust only. So, the soundness PBT and formatting targets are run with `enable_is: true`

Reply:
I see.